src/legacy/components/CurrencyInputPanel/FiatValue/FiatValueMod.tsx

```diff
@@ -1,67 +0,0 @@
-// eslint-disable-next-line no-restricted-imports
-import { t } from '@lingui/macro'
-import { Currency, CurrencyAmount, Percent } from '@uniswap/sdk-core'
-import { useMemo } from 'react'
-
-import useTheme from 'legacy/hooks/useTheme'
-import { ThemedText } from 'legacy/theme'
-import { warningSeverity } from 'legacy/utils/prices'
-import { MouseoverTooltip } from 'legacy/components/Tooltip'
-
-// MOD imports
-// mod
-import Loader from 'legacy/components/Loader'
-import { formatPercent } from 'utils/amountFormat'
-import { FiatAmount } from 'common/pure/FiatAmount'
-
-export function FiatValue({
```

Comment:
All good, not need to addres, but its a pitty git don't detected this as a move (shows as file added and removed)

not sure if `git mv` would have helped here. Or do one commit with the moving of the file, another with the changes

Reply:
Actually, I moved them using `git mv`. I guess it doesn't detect it properly because there are lots of changes